concent_api/concent_api/middleware.py

```diff
@@ -1,8 +1,19 @@
+import logging
 import os
+import sys
+import traceback
 
 from django.conf    import settings
+from django.db import transaction
+from django.http import HttpResponseServerError
+from django.http import JsonResponse
 
 from golem_messages import __version__
+from concent_api.constants import DEFAULT_ERROR_MESSAGE
+from utils.constants import ErrorCode
+
+
+logger = logging.getLogger('django.request')
```

Comment:
Please call it something else, like `request_logger`.

It's typical in our code to do `logger = logging.getLogger(__name__)` at the top of the module. So the natural assumption is that a variable called `logger` refers to the default logger in a module. If someone comes here later and changes something, he'll likely assume that it's just such a logger without even checking it.

Since `django.request` behaves in a very special way, we must make sure it's not overused. For that reason it should not be confused with a normal logger. Maybe even it would be better to call `getLogger()` only in the function where you're using it instead of putting it here, at the module level.

Reply:
done